### ISSUE TITLE
Cannot find module '/var/folders/...../CodePushResourcesMap.json' on android builds

### ISSUE THREAD

**Issue by orzarchi** [Microsoft/react-native-code-push#425]
Hiya
Since adding "../../node_modules/react-native-code-push/android/codepush.gradle" to my gradle config, my builds keep failing with that error due to the task "generateBundledResourcesHashRelease".

Here's the stack:
`Error: Cannot find module '/var/folders/0q/18m7xx1x5mv5g3p5d5v8v6dw0000gn/T/CodePushResourcesMap.json'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> ({PATH}/node_modules/react-native-code-push/scripts/generateBundledResourcesHash.js:32:32)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
`

Am I doing something wrong?
OS: OSX 10.11.5 El capitan
Node: 4.4.3

**Comment by geof90** [comment]
@orzarchi This should not happen unless

1. The `recordFilesBeforeBundleCommand` build step did not get run
2. The temp directory is cleared in some other build step removing that file.

Could you paste the output of `react-native run-android` so i can better see what's going on with your build?

**Comment by orzarchi** [comment]
I've managed to solve the problem thanks to your hints.
I've noticed that the `recordFilesBeforeBundleCommand` build step was indeed not being called, so I looked at my gradle files.

I've noticed I was applying the codepush gradle file before applying the react one, so I tried changing the order as follows:

```
apply from: "react.gradle"
apply from: "../../node_modules/react-native-code-push/android/codepush.gradle"
```

And now the recordFilesBeforeBundleCommandRelease step was called and the build passed.

Cheers!
